decidim-core/app/views/decidim/endorsements/_identity.html.erb

```diff
@@ -1,4 +1,4 @@
-<button type="button" class="<%= selected ? "is-selected" : "" %>" data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
+<button type="button" <%= selected ? 'class="is-selected"'.html_safe : "" %> data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
```

Comment:
I see that we usually do this in other parts of the application: 

```suggestion
<button type="button" class="<%= "is-selected" if selected %>" data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
```

Reply:
This is what i get if i apply the suggestion: 
![image](https://github.com/decidim/decidim/assets/105683/cac08a7e-2a6b-4487-a68e-001b3042db26)